### ISSUE TITLE
Unexpected status code - 503

### ISSUE THREAD

**Issue by psousal** [danmunn/redmine_mylyn_connector#18]
Hello,

I have redmine 2.2.3 and eclipse 4.2 Juno versions, and i have install plgins in eclipse and redmine, but when i try to connect the repository, in validate settings, i have this error "unexperted status code -503"

any idea?

**Comment by cforce** [comment]
Which update site did you use?

http://redmin-mylyncon.sourceforge.net/update-site/N/  ?

**Comment by psousal** [comment]
Hi Terence,

Thank you for reply,

Yes, I am using this update. I don’t know what it is happened…  I have web access but when I try to connect via eclipse I have the error.

I show you this screenshot:

[cid:image007.jpg@01CE5214.C1253DF0]

[cid:image008.jpg@01CE5214.C1253DF0]

[cid:image009.jpg@01CE5214.C1253DF0]

[cid:image010.jpg@01CE5214.C1253DF0]

De: Terence Miller [mailto:notifications@github.com]
Enviado el: miércoles, 15 de mayo de 2013 20:44
Para: danmunn/redmine_mylyn_connector
CC: Sousa Lastra Pablo
Asunto: Re: [redmine_mylyn_connector] Unexpected status code - 503 (#18)

Which update site did you use?

http://redmin-mylyncon.sourceforge.net/update-site/N/ ?

—
Reply to this email directly or view it on GitHubhttps://github.com/danmunn/redmine_mylyn_connector/issues/18#issuecomment-17957960.
Antes de imprimir este mensaje o sus documentos anexos, asegúrese de que es necesario. Proteger el medio ambiente está en nuestras manos.

Before printing this e-mail or attachments, be sure it is necessary. It is in our hands to protect the environment.
Antes de imprimir este mensaje o sus documentos anexos, asegúrese de que es necesario. Proteger el medio ambiente está en nuestras manos.

Before printing this e-mail or attachments, be sure it is necessary. It is in our hands to protect the environment.

*****************_AVISO LEGAL**_*******************
Este mensaje es privado y confidencial y solamente para la persona a la que va dirigido. Si usted ha recibido este mensaje por error, no debe revelar, copiar, distribuir o usarlo en ningún sentido. Le rogamos lo comunique al remitente y borre dicho mensaje y cualquier documento adjunto que pudiera contener. No hay renuncia a la confidencialidad ni a ningún privilegio por causa de transmisión errónea o mal funcionamiento.
Cualquier opinión expresada en este mensaje pertenece únicamente al autor remitente, y no representa necesariamente la opinión de Grupo Santander, a no ser que expresamente se diga y el remitente esté autorizado para hacerlo. Los correos electrónicos no son seguros, no garantizan la confidencialidad ni la correcta recepción de los mismos, dado que pueden ser interceptados, manipulados, destruidos, llegar con demora, incompletos, o con virus. Grupo Santander no se hace responsable de las alteraciones que pudieran hacerse al mensaje una vez enviado.
Este mensaje sólo tiene una finalidad de información, y no debe interpretarse como una oferta de venta o de compra de valores ni de instrumentos financieros relacionados. En el caso de que el destinatario de este mensaje no consintiera la utilización del correo electrónico vía Internet, rogamos lo ponga en nuestro conocimiento.

*********************_DISCLAIMER**_**************
This message is private and confidential and it is intended exclusively for the addressee. If you receive this message by mistake, you should not disseminate, distribute or copy this e-mail. Please inform the sender and delete the message and attachments from your system. No confidentiality nor any privilege regarding the information is waived or lost by any mistransmission or malfunction.
Any views or opinions contained in this message are solely those of the author, and do not necessarily represent those of Grupo Santander, unless otherwise specifically stated and the sender is authorized to do so. E-mail transmission cannot be guaranteed to be secure, confidential, or error-free, as information could be intercepted, corrupted, lost, destroyed, arrive late, incomplete, or contain viruses. Grupo Santander does not accept responsibility for any changes in the contents of this message after it has been sent.
This message is provided for informational purposes and should not be construed as a solicitation or offer to buy or sell any securities or related financial instruments. If the addressee of this message does not consent to the use of internet e-mail, please communicate it to us.

**Comment by cforce** [comment]
Sorry can't see any screenhsot, only placeholder

**Comment by psousal** [comment]
Hello Terence, 

Attached i send the screenshot

**Comment by psousal** [comment]
![captura1](https://f.cloud.github.com/assets/4427323/512668/da765f10-be2e-11e2-873d-559a0104c40c.png)
![captura2](https://f.cloud.github.com/assets/4427323/512669/da8cedca-be2e-11e2-974b-8a8084aab6a9.png)
![captura3](https://f.cloud.github.com/assets/4427323/512670/da9f431c-be2e-11e2-966e-daa77de93801.png)
![captura4](https://f.cloud.github.com/assets/4427323/512671/daa29db4-be2e-11e2-9039-2872196e3171.png)

**Comment by psousal** [comment]
And this is my environment:

Environment:
  Redmine version                          2.2.3.stable
  Ruby version                             1.9.3 (x86_64-linux)
  Rails version                            3.2.12
  Environment                              production
  Database adapter                         Mysql2
Redmine plugins:
  redmine_dmsf                             1.4.5p1 stable
  redmine_information                      1.0.2
  redmine_people                           0.1.6

**Comment by cforce** [comment]
Did you activate the "REST api" in redmine admin authenification settings view?
You shall see xml site with issues for http://180.88.66.48/redmine/issues.xml

**Comment by psousal** [comment]
Yes, I have activated Rest api in authentification settings and I can see this xml without any problems…

De: Terence Miller [mailto:notifications@github.com]
Enviado el: jueves, 16 de mayo de 2013 15:57
Para: danmunn/redmine_mylyn_connector
CC: Sousa Lastra Pablo
Asunto: Re: [redmine_mylyn_connector] Unexpected status code - 503 (#18)

Did you activate the "REST api" in redmine admin authenification settings view?
You shall see xml site with issues for http://180.88.66.48/redmine/issues.xml

—
Reply to this email directly or view it on GitHubhttps://github.com/danmunn/redmine_mylyn_connector/issues/18#issuecomment-18002399.
Antes de imprimir este mensaje o sus documentos anexos, asegúrese de que es necesario. Proteger el medio ambiente está en nuestras manos.

Before printing this e-mail or attachments, be sure it is necessary. It is in our hands to protect the environment.
Antes de imprimir este mensaje o sus documentos anexos, asegúrese de que es necesario. Proteger el medio ambiente está en nuestras manos.

Before printing this e-mail or attachments, be sure it is necessary. It is in our hands to protect the environment.

*****************_AVISO LEGAL**_*******************
Este mensaje es privado y confidencial y solamente para la persona a la que va dirigido. Si usted ha recibido este mensaje por error, no debe revelar, copiar, distribuir o usarlo en ningún sentido. Le rogamos lo comunique al remitente y borre dicho mensaje y cualquier documento adjunto que pudiera contener. No hay renuncia a la confidencialidad ni a ningún privilegio por causa de transmisión errónea o mal funcionamiento.
Cualquier opinión expresada en este mensaje pertenece únicamente al autor remitente, y no representa necesariamente la opinión de Grupo Santander, a no ser que expresamente se diga y el remitente esté autorizado para hacerlo. Los correos electrónicos no son seguros, no garantizan la confidencialidad ni la correcta recepción de los mismos, dado que pueden ser interceptados, manipulados, destruidos, llegar con demora, incompletos, o con virus. Grupo Santander no se hace responsable de las alteraciones que pudieran hacerse al mensaje una vez enviado.
Este mensaje sólo tiene una finalidad de información, y no debe interpretarse como una oferta de venta o de compra de valores ni de instrumentos financieros relacionados. En el caso de que el destinatario de este mensaje no consintiera la utilización del correo electrónico vía Internet, rogamos lo ponga en nuestro conocimiento.

*********************_DISCLAIMER**_**************
This message is private and confidential and it is intended exclusively for the addressee. If you receive this message by mistake, you should not disseminate, distribute or copy this e-mail. Please inform the sender and delete the message and attachments from your system. No confidentiality nor any privilege regarding the information is waived or lost by any mistransmission or malfunction.
Any views or opinions contained in this message are solely those of the author, and do not necessarily represent those of Grupo Santander, unless otherwise specifically stated and the sender is authorized to do so. E-mail transmission cannot be guaranteed to be secure, confidential, or error-free, as information could be intercepted, corrupted, lost, destroyed, arrive late, incomplete, or contain viruses. Grupo Santander does not accept responsibility for any changes in the contents of this message after it has been sent.
This message is provided for informational purposes and should not be construed as a solicitation or offer to buy or sell any securities or related financial instruments. If the addressee of this message does not consent to the use of internet e-mail, please communicate it to us.

**Comment by maxrossello** [comment]
Same here, but with a 404 over redmine 2.3.
I tried the verification url directly in the browser: mylyn/version?key=12ffe401c3d76a4edxxxxx...

Plugin routes are correctly identified, but it seems that controllers are not reached.
